cloudbuild.yaml

```diff
@@ -0,0 +1,14 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+timeout: 1200s
+steps:
+  - name: 'gcr.io/jason-heptio/gcb-docker-gcloud'
```

Comment:
eventually this would be one of the staging repos?

Reply:
Yes, went ahead and marked as WIP until https://github.com/kubernetes/test-infra/pull/14219 merges and we can use that image instead.